carbonmark-api/helpers/fetchProjectDetails/index.js

```diff
@@ -0,0 +1,21 @@
+import { fetchGSProject } from "./fetchGSProject";
+import { fetchVCSProject } from "./fetchVCSProject";
+
+export const fetchProjectDetails = async ({ key }) => {
+  try {
+    const [registry, registryProjectId] = key.split("-");
+    if (!registry || !registryProjectId) return null;
+
+    const registryUpper = registry.toUpperCase();
+    if (registryUpper === "VCS") {
```

Comment:
Without having to go down the refactor road too much it would be nice to have these magic strings as constants somewhere